Gemfile

```diff
@@ -1,21 +1,18 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-  "https://github.com/#{repo_name}.git"
-end
-
+ruby '2.5.1'
```

Comment:
We already have this line down in the custom section.
Let's kill line 64 then and keep the default one up here.